Jenkinsfile

```diff
@@ -23,9 +23,8 @@ pipeline {
           withCredentials(bindings: [string(credentialsId: 'FabricApiKey', variable: 'FABRIC_API_KEY'), string(credentialsId: 'FabricBuildSecret', variable: 'FABRIC_BUILD_SECRET')]) {
             sh '/usr/local/bin/pod install'
             sh 'set -o pipefail && xcodebuild -workspace Yona.xcworkspace -scheme Yona -sdk iphonesimulator -destination \'platform=iOS Simulator,name=iPhone 6,OS=11.4\' -derivedDataPath ./BuildOutput clean build test | /usr/local/bin/xcpretty --report junit --output ./BuildOutput/Report/testreport.xml'
-            sh '''set +x
-"Pods/Fabric/run" "$FABRIC_API_KEY" "$FABRIC_BUILD_SECRET"
- 			   '''
+            sh 'set +x'
```

Comment:
This is a bad commit: the ``set +x`` should be in the same ``sh`` statement, as it otherwise as null effect, causing the secret to be revealed. The original construct is the only right one:

```
sh '''set +x
"Pods/Fabric/run" "$FABRIC_API_KEY" "$FABRIC_BUILD_SECRET"
'''
```

Reply:
Updated.